src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -243,10 +243,16 @@ private Run CreateSarifRun()
         {
             if (!string.IsNullOrWhiteSpace(originalUriBasePath))
             {
-                if (originalUriBasePath.StartsWith("/") &&
-                     platform == "Linux")
+                if (originalUriBasePath.StartsWith("/"))
                 {
-                    originalUriBasePath = "file://" + originalUriBasePath;
+                    if (platform == "Linux")
+                    {
+                        originalUriBasePath = "file://" + originalUriBasePath;
+                    }
+                    else
+                    {
+                        return null;
+                    }
```

Comment:
is this right? if we have platform == windows, u are returning null and not changing  the originalUriBasePath. If we compare with the original file, it was always executing the following lines.

Reply:
I think the original file have defect, 
the below lines will always look to create a UriKind.Absolute from it,
if the platform == windows and the uri starts with /, it is not possible a absolute uri, it is a relative web path.

the way it works was, since we test the tool only in windows, the Uri.TryCreate will return false, and the last line will return null, which is correct.

but when running in Linux, the Uri.TryCreate will actually return true and return a File path using the relative web path.

this added condition check prevented that.